### PR TITLE
feature: Return an array of messages from server. JS fetchs that data…

### PR DESCRIPTION
**Pull request by LindaNayeli104**
… and adds a random massage to the page

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java [hunk 1]
```diff
@@ -5,14 +5,30 @@
...
+    // Convert the server stats to JSON
+    String json = convertToJsonUsingGson(bucketListTrips);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+    
+  }
+
+  private String convertToJsonUsingGson(String[] bucketListTrips) {
```
Comment (papakoj) [inline portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java:29]: Good work making a helper method

File: portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java [hunk 2]
```diff
@@ -5,14 +5,30 @@
...
 public class MyServletClass extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello world, I'm Linda!");
+    String[] bucketListTrips = {"India", "Italy", "Thailand"};
+
+    
+
```
Comment (papakoj) [inline portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java:19]: Nit (short for nitpick): Please remove these empty lines. You can leave one if it makes it easier to read.

**Review by papakoj — APPROVED** [review]
Nice work! Please respond to the comments before merging :)

File: portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java (hunk 1 above)
Comment (LindaNayeli104) [reply portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java:29]: Thanks!

File: portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java (hunk 2 above)
Comment (LindaNayeli104) [reply portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java:19]: Done!  Thanks for the observation!

**Pull request merged**